README.md

```diff
@@ -89,6 +89,14 @@ In **./notebooks/generative_module** there is a set of notebooks exploring how t
 
 Note that a seed has not been implemented to reproduce the outputs shown.
 
+###  License Agreements
+
```

Comment:
This is good to let the user know how to sign up but I am wondering if we need to make it a little more formal:

## Licensing

This project uses LLaMA 2 and LLaMA 3 models from Meta AI. Usage of these models is governed by the non-commercial license provided by Meta. 

### LLaMA 2

LLaMA 2 is licensed under Meta's non-commercial license. For more details, please refer to [Meta's LLaMA 2 licensing page](https://example.com/llama2-license). To sign the corresponding license agreement you can apply for this via huggingface[https://huggingface.co/meta-llama/Llama-2-70b].

### LLaMA 3

LLaMA 3 is also licensed under Meta's non-commercial license. For more details, please refer to [Meta's LLaMA 3 licensing page](https://example.com/llama3-license). To sign the corresponding license agreement you can apply for this via huggingface[https://huggingface.co/meta-llama/Meta-Llama-3-70B].

By using these models in this project, you agree to comply with the licensing terms provided by Meta.

For any commercial use or further inquiries, please contact Meta AI directly.